### ISSUE TITLE
Dynamic reading in Windows Terminal

### ISSUE THREAD

**Issue by jeanbrazcosta** [nvaccess/nvda#13075]
NVDA is not dynamic reading in Windows Terminal after a few lines.

When opening Windows terminal, after having several lines in the terminal, when entering any command that returns some information, NfVDA does not read.
### Steps to reproduce:
Open Windows terminal and use git for example or whatever, use several commands after a while NVDA will stop announcing dynamic changes
### Actual behavior:
NVDA stops reading
### Expected behavior:
Deveria anunciar as informações retornardas após algum comando, exemplo, git checkout branch
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
2021.2
#### Windows version:
21 h 1
#### Name and version of other software in use when reproducing the issue:
Windows Terminal version: 1.0

**Comment by lukaszgo1** [comment]
cc @codeofdusk

**Comment by codeofdusk** [comment]
When NVDA stops reading in Windows Terminal, does Windows Explorer continue to work as intended, or do parts of the interface suddenly become inaccessible (i.e. focus tracking becomes unreliable)?

**Comment by Neurrone** [comment]
Unfortunately I encounter this on a daily basis, see microsoft/terminal/#10911. It doesn't seem to be high on their priority of things to fix though unfortunately.

**Comment by jeanbrazcosta** [comment]
Yes, when the aniver stops reading the focus still remains reliable.

Enviado do meu iPhone

**Comment by codeofdusk** [comment]
Duplicate of microsoft/terminal#10911. Sadly I haven't been able to reproduce this.

**Comment by Neurrone** [comment]
I'm using Terminal 1.11.2921.0, Windows 10.0.19043.1348.

**Comment by Neurrone** [comment]
I also forgot to mention that nearly all of my usage of Terminal is either SSH or WSL2. In case this helps with reproducing the problem.

**Comment by Adriani90** [comment]
The referenced terminal bug has been fixed in January 2023, and the issue is not reproducible anymore with NvDA 2024.4.1. Closing as works for me. If you are still having this issue, pleaes comment and we can reopen.
